### DISCUSSION TITLE
docker-compose instead docker compose

### DISCUSSION THREAD

**Discussion by XDavidT** [NetApp/harvest#1701] [Q&A] [answered]
I saw documents are still showing docker-compose commands, while docker not supported anymore with this.
You should update it into `docker compose`

**Comment by cgrinds** [comment, marked as answer]
Thanks @XDavidT! Originally the documentation only used `docker compose`. We reverted to `docker-compose` because many of our customers had not upgraded to a recent version of Docker (or their package manager hadn't) and it only caused confusion.

We should revisit or make it clearer in the documentation that you should try one or other depending on your docker version. Thanks!
